native/src/main/native/org/mozilla/jss/ssl/SSLCipher.c

```diff
@@ -19,7 +19,14 @@ static const CK_MECHANISM_TYPE auth_alg_defs[] = {
     CKM_RSA_PKCS,          /* ssl_auth_rsa_sign */
     CKM_RSA_PKCS_PSS,      /* ssl_auth_rsa_pss */
     CKM_NSS_HKDF_SHA256,   /* ssl_auth_psk (just check for HKDF) */
+#ifndef NSS_VERSION_PQC_DEF
     CKM_INVALID_MECHANISM  /* ssl_auth_tls13_any */
+#else
+    CKM_INVALID_MECHANISM,  /* ssl_auth_tls13_any */
+    CKM_ML_DSA,            /* ssl_auth_mldsa (same mech for each) */
+    CKM_ML_DSA,            /* ssl_auth_mldsa (key determines the*/
+    CKM_ML_DSA            /* ssl_auth_mldsa  parameter set) */
+#endif
```

Comment:
Just for reference, this change corresponds to an NSS patch that currently exists on CentOS 10 only:
https://gitlab.com/redhat/centos-stream/rpms/nss/-/blob/c10s/nss-3.112-add-ml-dsa-ssl-support.patch?ref_type=heads#L965